parser-typechecker/src/Unison/Codebase/Editor/RemoteRepo.hs

```diff
@@ -99,10 +124,10 @@ maybePrintPath path =
     then mempty
     else "." <> Path.toText path
 
-data ReadRemoteNamespace
+data ReadRemoteNamespace server
```

Comment:
This change (and corresponding on `ReadShareRemoteNamespace`, `WriteRemotePath`, `WriteShareRemotePath`) seems a little weird to me.

I guess it is a type that evolves during discovery from one parameter to another? But maybe it should be separate from this data structure that has to do with Git too.

Reply:
I don't personally find it confusing to have parameters on types like this, I usually just jump-to-definition and see what it's used for, or ignore it entirely if I don't need to interact with it; I tend to find the proliferation of multiple 'identical except for this one deeply nested change' types more confusing, but maybe that's just me.

In this case I'm especially not worried since Share is the new "standard" and git will be slowly phased out over time.